cookbooks/convection-box.prm

```diff
@@ -101,7 +101,9 @@ end
 
 subsection Formulation
   set Formulation = custom
-  set Temperature density approximation = adiabatic
+  set Mass conservation    = incompressible
```

Comment:
Why do you change it at all?

I'm ok with just removing the whole block. That's what the setup was before Nov anyway, right?